lib/Doctrine/Annotations/Type/Type.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Annotations\Type;
+
+/**
+ * @internal
+ */
+interface Type
+{
+    public function describe() : string;
```

Comment:
A description of the intent of this method is needed. What is the "description" you will get from this? Human-readable? Machine-friendly? An identifier? The type FQN?